stl/inc/format

```diff
@@ -1496,6 +1496,214 @@ public:
     }
 };
 
+#pragma warning(push)
+#pragma warning(disable : 4265) // non-virtual destructor in base class

```

Comment:
I think the compiler is smart enough not to warn if we explicitly default a protected destructor.

Reply:
Good, replaced this with a defaulted dtor.